src/pos/client.ts

```diff
@@ -281,6 +288,26 @@ export class PoolProtocolImpl extends AbstractProtocolClient {
                     unstake
                 )} CTSI, withdraw ${formatCTSI(withdraw)} CTSI`
             );
+
+            if (this.lastRebalanceTimestamp + this.rebalanceIntervalMillis > Date.now()) {
+                log.info(
+<<<<<<< HEAD
```

Comment:
You need to properly squash the commit and remove this conflict.

Reply:
Yup sorry, I'm too used to phabricator. Fixing it